creator-node/src/logging.js

```diff
@@ -21,12 +21,15 @@ function requestNotExcludedFromLogging (url) {
 }
 
 function getRequestLoggingContext (req) {
+  req.startTime = process.hrtime()
   const requestID = shortid.generate()
+  const urlParts = req.url.split('?')
   return {
     requestID: requestID,
     requestMethod: req.method,
     requestHostname: req.hostname,
-    requestUrl: req.originalUrl,
+    requestUrl: urlParts[0],
+    requestQueryParams: urlParts.length > 1 ? urlParts[1] : undefined,
```

Comment:
this is great. i don't think this will handle the /ipfs/ routes since they're not query params, but i don't think that's necessary right now. i think this is a huge win